tests/end-to-end/run.sh

```diff
@@ -0,0 +1,262 @@
+#!/bin/bash
+
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+
+set -e
+env | grep "PATH"
+
+# End-to-end testing script.
+
+verify_diff () {
```

Comment:
let's move this to the `verify_diff.py` script as well, so we have all forms of verification encapsulated in a single script.

Reply:
done.